lib/Common/Codex/Utf8Helper.h

```diff
@@ -72,6 +77,7 @@ namespace utf8
         }
```

Comment:
Nit: the function signature is a mix of old-style and new-style SAL annotation. Additionally, is it \_In\_reads\_(sourceCount) sourceString?

Reply:
That sounds correct, it does read at-most `sourceCount` wide chars from `sourceString`. If you do change this, then you should also change `WideStringToNarrow` above too.